frontend/src/features/projects/common/forms/AppraisalCheckListForm.tsx

```diff
@@ -22,26 +21,21 @@ const AppraisalCheckListForm: React.FunctionComponent<IAppraisalCheckListFormPro
     statusCode: props.taskStatusCode ?? ReviewWorkflowStatus.AppraisalReview,
   });
   return (
-    <Container fluid className={classNames(props.className)}>
```

Comment:
Causing formatting inconsistencies

Reply:
Have you double checked that no parent is using this class name prop? If you wanted to keep this more equivalent but remove the container you could add the classname to the row.